docs/topics/managedcluster.md

```diff
@@ -0,0 +1,147 @@
+# Managed Clusters (AKS)
+- **Feature status:** Experimental
+- **Feature gate:** AKS=true,MachinePool=true
+
+Cluster API Provider Azure (CAPZ) experimentally supports managing Azure
+Kubernetes Service (AKS) clusters. CAPZ implements this with three
+custom resources:
+- AzureManagedControlPlane
+- AzureManagedCluster
+- AzureManagedMachinePool
+
+The combination of AzureManagedControlPlane/AzureManagedCluster
+corresponds to provisioning an AKS cluster. AzureManagedMachinePool
+corresponds one-to-one with AKS node pools. This also means that
+creating an AzureManagedControlPlane requires defining the default
+machine pool, since AKS requires at least one system pool at creation
+time.
+
+## Deploy with clusterctl
+
+A clusterctl flavor exists to deploy an AKS cluster with CAPZ. This
+flavor requires the following environment variables to be set before
+executing clusterctl.
+
+```bash
+# Kubernetes values
+export CLUSTER_NAME="foobar"
+export WORKER_MACHINE_COUNT=2
+export KUBERNETES_VERSION="1.17.4" # NOTE: no preceding "v", unlike normal for CAPI.
```

Comment:
why is that? is there anyway we can standardize with CAPI and trim the v when using the version? especially since we're reusing the same env var

Reply:
also the example template version below has a v prefix https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/686/files#diff-96e57bd2edb999ae76c554c70314028fR85

Reply:
because AKS, but we can change the code to trim as you suggest, sure. and whoops